nmap-payloads

```diff
@@ -315,3 +324,78 @@ udp 9987
 # https://github.com/memcached/memcached/blob/master/doc/protocol.txt
 udp 11211
 "\0\x01\0\0\0\x01\0\0version\r\n"
+
+
+# ECHO RAPID7
+udp 7
```

Comment:
Do we need this if there is an existing payload for udp 7? https://github.com/rkirk-r7/nmap/blob/12c9b61ad242688f7350c8940868ac89e80d79b2/nmap-payloads#L36-L37